xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/accounts/AccountTransactionsRequestParams.java

```diff
@@ -38,10 +38,70 @@
    *
    * @return An {@link ImmutableAccountTransactionsRequestParams.Builder}.
    */
+  @Deprecated
```

Comment:
Let's add a message that clarifies why this is being deprecated, maybe something like this suggestion.
```suggestion
  @Deprecated Prefer one of the other defined builders so that you are overtly specifying which ledger index or type you want.
```

@nkramer44 Any thoughts here around better wording?

Reply:
I think that wording is fine.